refactor/data/app/api/api_v0/endpoints/fs.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2021 Agnostiq Inc.
+#
+# This file is part of Covalent.
+#
+# Licensed under the GNU Affero General Public License 3.0 (the "License").
+# A copy of the License may be obtained with this software package or at
+#
+#      https://www.gnu.org/licenses/agpl-3.0.en.html
+#
+# Use of this file is prohibited except in compliance with the License. Any
+# modifications or derivative works of this file must retain this copyright
+# notice, and modified files must contain a notice indicating that they have
+# been altered from the originals.
+#
+# Covalent is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the License for more details.
+#
+# Relief from the License may be granted by purchasing a commercial license.
+
+
+from typing import Any
+
+from app.schemas.common import HTTPExceptionSchema
+from app.schemas.fs import UploadResponse
+from fastapi import APIRouter, File, HTTPException
+from fastapi.responses import FileResponse
+
+router = APIRouter()
+
+
+@router.post("/upload", status_code=200, response_model=UploadResponse)
+def upload_file(*, file: bytes = File(...)) -> Any:
```

Comment:
According to the FastAPI docs I believe it should be set to `bytes` if we want to have the file contents in memory, however I am not totally familiar with the FileStorage object you mentioned. The docs also mention the `UploadFile` type annotation which uses a `Spoofed File`, which is described as "A file stored in memory up to a maximum size limit, and after passing this limit it will be stored in disk." - https://fastapi.tiangolo.com/tutorial/request-files/#file-parameters-with-uploadfile

Reply:
ok cool, if that's what the FastAPI docs say then let's do it